synce_clock.c

```diff
@@ -285,3 +289,58 @@ int synce_clock_poll(struct synce_clock *clk)
 
 	return ret;
 }
+
+int synce_clock_check_dev(struct synce_clock *clk, char *dev_name)
```

Comment:
It can't be done. The code does not compile, since synce_clock.h is not aware of struct synce_dev. I think that this separation to layers was done intentionally.
From reading the code, it should not be done this way. Do you think I should include synce_dev.h in synce_clock.h?

Reply:
you can forward declare the type and use only pointer, it doesn't have to be defined to be used as parameter.

Reply:
fixed it accordingly